pkg/csi/service/common/constants.go

```diff
@@ -341,6 +341,12 @@ const (
 	// CreateCSINodeAnnotation is the annotation applied by spherelet
 	// to convey to CSI driver to create a CSINode instance for each node.
 	CreateCSINodeAnnotation = "vmware-system/csi-create-csinode-object"
+
+	// KubeSystemNamespace is the namespace for system resources.
+	KubeSystemNamespace = "kube-system"
+
+	// WCPFSSConfigMapName is the name of the configmap where WCP component's FSS values are stored.
+	WCPFSSConfigMapName = "wcp-cluster-capabilities"
```

Comment:
can we change this name to `WCPCapabilityConfigMapName`

Reply:
+1

Reply:
Done